mediator/esmFldsExchange_cesm_mod.F90

```diff
@@ -435,52 +464,42 @@ subroutine esmFldsExchange_cesm(gcomp, phase, rc)
     ! to lnd: ice sheet mask where we are potentially sending non-zero fluxes from glc
     ! to lnd: fields with multiple elevation classes from glc
     ! ---------------------------------------------------------------------
-    allocate(flds(2))
-    flds = (/'Sg_icemask               ', 'Sg_icemask_coupled_fluxes'/)
-
-    do n = 1,size(flds)
-       fldname = trim(flds(n))
-       if (phase == 'advertise') then
-          call addfld(fldListFr(compglc)%flds   , trim(fldname))
-          call addfld(fldListTo(complnd)%flds   , trim(fldname))
-       else
-          if ( fldchk(is_local%wrap%FBExp(complnd)         , trim(fldname), rc=rc) .and. &
-               fldchk(is_local%wrap%FBImp(compglc, compglc), trim(fldname), rc=rc)) then
-             call addmap(fldListFr(compglc)%flds, trim(fldname), complnd,  mapconsd, 'one', glc2lnd_smap)
-             call addmrg(fldListTo(complnd)%flds, trim(fldname), &
-                  mrg_from1=compglc, mrg_fld1=trim(fldname), mrg_type1='copy')
-          end if
-       end if
-    end do
-    deallocate(flds)
 
-    ! for glc fields with multiple elevation classes in glc->lnd
+    ! The suffix _elev on land fields designates fields with elevation classes
```

Comment:
Can you consider different wording here? "BROKEN" in all caps is a little disturbing until you read it several times.

Reply:
Good point. Done.